### DISCUSSION TITLE
How to Search for AVL inside my Codespace

### DISCUSSION THREAD

**Discussion by Mafia-1911** [community/community#142725] [Codespaces]
### Select Topic Area

Question

### Body

How to Search for AVL inside my Codespace

**Comment by Mafia-1911** [comment]
Just go to the Abouts Tab and look for AVL tree , Then File -> look For- Enter the name and you find your desired AVL

**Comment by Mafia-1911** [comment]
Just go to the Abouts Tab and look for AVL tree , Then File -> look For- Enter the name and you find your desired AVL

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/142727#discussioncomment-11078546) for reference to why this discussion was closed and locked. Thanks!
